packages/amazon-bedrock/src/bedrock-embedding-model.test.ts

```diff
@@ -86,14 +86,6 @@ describe('doEmbed', () => {
     expect(usage?.tokens).toStrictEqual(8);
   });
 
-  it('should handle multiple input values and extract usage', async () => {
-    const { usage } = await model.doEmbed({
-      values: testValues,
-    });
-
-    expect(usage?.tokens).toStrictEqual(16);
-  });
```

Comment:
why was  this test removed?

Reply:
removed because we now throw an error 👍 
![image](https://github.com/user-attachments/assets/587a8618-8540-4626-b3c1-1d955c3aca38)